src/Components/Product/products.css

```diff
@@ -0,0 +1,114 @@
+.grid-container {
+  display: grid;
+  grid-template-columns: 21vw 1fr;
+  column-gap: 1.5rem;
+}
+
+/* product list  box*/
+.products-category-box {
+  overflow-x: hidden;
+}
+/* filter list  styling*/
+.filter-list {
+  padding: 1rem;
+  height: calc(100vh - 6rem);
+  overflow: scroll;
+  position: sticky;
+  top: 9%;
+  left: 1%;
+  border: 1px solid var(--border-shade);
+  border-right-color: var(--theme-shade-3);
+  border-left: unset;
+}
+.btn-clear {
+  padding: 4px;
+  letter-spacing: 0.8px;
+  float: right;
+}
+input[type="radio"],
+input[type="checkbox"] {
+  accent-color: var(--theme-color);
+}
+.filter-list .flex-column {
+  justify-content: space-between;
+  padding: 1rem;
+  gap: 0.5rem;
+}
+
+/* products styling */
+.products-list {
+  flex-wrap: wrap;
+  gap: 1rem;
+  margin-bottom: 5rem;
+}
+.card-badge {
+  left: 0%;
+  top: 2%;
+}
+.card-badge::after {
+  border-width: 9.2px 5px;
+  border-color: rgba(218, 47, 41, 0.55) transparent rgba(218, 47, 41, 0.55)
+    rgba(218, 47, 26, 0.7);
+}
+
+.out-stock-container {
+  border-bottom-right-radius: 1rem;
+  border-top-left-radius: 1rem;
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
```

Comment:
I think there is a shorthand for these four properties. 
You can use `inset:0;` instead of using all four individually.

Reply:
I will change it